package.json

```diff
@@ -128,7 +128,9 @@
     "update-mock-cdn": "node test/e2e/mock-cdn/update-mock-cdn-files.js",
     "update-snap-binary": "tsx test/e2e/mock-response-data/snaps/update-snap-binary.ts",
     "attributions:check": "./development/attributions-check.sh",
-    "attributions:generate": "./development/generate-attributions.sh"
+    "attributions:generate": "./development/generate-attributions.sh",
+    "yarn-binary:download": "corepack pack -o .yarn/yarn-corepack.tgz",
+    "yarn-binary:hydrate": "corepack hydrate .yarn/yarn-corepack.tgz --activate"
```

Comment:
We'll need to call this in CI for this to have the intended impact, right? I guess that would happen in `github-tools`

Reply:
Yes, I have a follow up task to work on this asap, thank you!!